agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1346,9 +1346,18 @@ bool backhaul_manager::backhaul_fsm_wireless(bool &skip_select)
 
             // Create a HAL instance if doesn't exists
             if (!soc->sta_wlan_hal) {
+
+                bwl::hal_conf_t hal_conf;
+
+                if (!beerocks::bpl::bpl_cfg_get_hostapd_ctrl_path(iface, hal_conf.wpa_ctrl_path)) {
```

Comment:
This should be the wpa_supplicant path, not hostapd path.

Reply:
fixed